tests/functional/features/cli/__snapshots__/update_snapshots.feature.snap

```diff
@@ -0,0 +1,7 @@
+
+
+exports[`Should echo salut1 1.1`] = `"salut1
```

Comment:
I don't think you're supposed to commit that either. Basically all snapshoting features tests are a bit messy like that.